### DISCUSSION TITLE
Which transformed plot looks "better" w/ varying cofactors?

### DISCUSSION THREAD

**Discussion by denvercal1234GitHub** [ImmuneDynamics/Spectre#95] [Q&A] [answered]
Hi there, 

The theory is very well described in the guideline (https://wiki.centenary.org.au/display/SPECTRE/Data+transformation) regarding which cofactor to use for the arcsinh transformation. 

Nevertheless, would you mind helping me assess visually which plots (Question 1) looks "better"? Is there sort of check list for what to ensure visually to ensure appropriate transformation, e.g., to have all the cells shown up on the plot, or negative population for a marker does not look too spread out visually, etc.?

Thank you again for your help! 

**Question 1.** 

<img width="785" alt="Screen Shot 2021-08-31 at 7 15 31 PM" src="https://user-images.githubusercontent.com/62179082/131554907-67b6a49f-dd12-4812-bfc3-e780dc8f1591.png">

This marker C seems to need a cofactor even higher than 1,000?
<img width="784" alt="Screen Shot 2021-08-31 at 7 19 56 PM" src="https://user-images.githubusercontent.com/62179082/131555549-a8d26e54-e224-4b61-abf0-ad66751faffc.png">

**Question 2.** In the figure below, cofactor = 100 seems to have 2 populations for CD8A, but cofactor = 1000 has these 2 populations merged. Which one in this case is more accurate/better?

<img width="811" alt="Screen Shot 2021-08-31 at 6 41 27 PM" src="https://user-images.githubusercontent.com/62179082/131550470-cd52526b-bf63-4fa7-8dd5-bffd4350c64e.png">

**Comment by SamGG** [comment, marked as answer]
IMHO
Q1: 300 sounds OK for all. 1000 is stacking events on the left.
Q2: well known effect of under-estimating the cofactor. A -1000, B 10000 (or a little bit less ~5000, in order to get similar dispersion of the positive and negative populations).
[Folcarelli et al. 2021](https://pubmed.ncbi.nlm.nih.gov/34327803/) gives an overview of the field. I like Finak et al. and Azad et al., but directly linked to your question is [this figure](https://journals.plos.org/plosone/article/figure?id=10.1371/journal.pone.0035693.g002) from Ray et Pyne in 2012.

**Reply by denvercal1234GitHub** [reply]
Thank you @SamGG  very much for your response! Do you in your analysis apply flowScape or OTflow or FlowVS (https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1083-9) in deciding cofactor within the Spectre workflow?
